src/AppInstallerCLIE2ETests/Interop/InstallInterop.cs

```diff
@@ -657,5 +657,73 @@ public void GetApplicableInstaller()
             Assert.AreEqual(PackageInstallerScope.User, packageInstallerInfo.Scope);
             Assert.AreEqual("en-US", packageInstallerInfo.Locale);
         }
+
+        /// <summary>
+        /// Install exe and verify that we can find it as installed after.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Test]
+        public async Task InstallExe_VerifyInstalledCatalog()
+        {
+            var installedCatalogReference = this.packageManager.GetLocalPackageCatalog(LocalPackageCatalog.InstalledPackages);
+
+            // Ensure package is not installed
+            string targetPackageProductCode = "{A499DD5E-8DC5-4AD2-911A-BCD0263295E9}";

```

Comment:
Nit: I had to look up what this GUID was. Can this be a constant somewhere else?